### DISCUSSION TITLE
不支持emoji吗? 😟

### DISCUSSION THREAD

**Discussion by cohhi** [CaiJimmy/hugo-theme-stack#817] [Q&A] [answered]
![1](https://user-images.githubusercontent.com/71576964/236428360-6cdda343-5c31-4812-a597-d1088197215a.PNG)
![2](https://user-images.githubusercontent.com/71576964/236428365-258e0463-6394-4de0-ad64-d701dca28330.PNG)

**Comment by rmdhnreza** [comment, marked as answer]
Make sure `enableEmoji: true` in `config.yaml`
![6zyc9QDtxd](https://user-images.githubusercontent.com/23427492/236596736-4f0563f4-0a92-4b4e-bb7c-449dfbb5bedf.png)
